test/system/012-manifest.bats

```diff
@@ -143,6 +143,29 @@ EOF
     validate_instance_compression "2" "$list" "amd64" "zstd"
     validate_instance_compression "3" "$list" "arm64" "zstd"
 
+    smanifest="$list"
+    run_podman manifest inspect --authfile=$authfile --tls-verify=false \
+        $manifestpushed
```

Comment:
Request: could you assign a new variable, maybe `pmanifest=$output`? That'll make it easier for someone reading line 164 below. It's also safer in case someone inadvertently adds a `run` command inside the loop.

For extra credit maybe, a few lines above, get rid of the ill-named `$list`, just make it `smanifest=$(skopeo ...)` from the first moment. Next person to look at that code will appreciate it.